src/rules/no-restricted-paths.js

```diff
@@ -6,6 +6,14 @@ import isStaticRequire from '../core/staticRequire'
 import docsUrl from '../docsUrl'
 import importType from '../core/importType'
 
+const allowedImportKindsSchema = {
+  type: 'array',
+  items: {
+    type: 'string',
+  },
+  uniqueItems: true,
+}
+
```

Comment:
i do think we should restrict this for now to known import types. we can extend it in the future if needed, but we can't lock it down so easily.